### DISCUSSION TITLE
Build failing for APK

### DISCUSSION THREAD

**Discussion by ritvij14** [subosito/flutter-action#125] [General]
https://github.com/ritvij14/cnc_flutter_app/runs/3425406195?check_suite_focus=true

I don't understand why it's failing, it was working fine a couple of days ago and running `flutter build apk` from my terminal makes the apk without any issues.

Please help. TIA

**Comment by uxduck** [comment]
Are you consciously using `flutter/2.5.0-5.2.pre-beta`?

**Comment by ritvij14** [comment]
@ben-xD actually I am using a file from what I found on an article of CI for flutter through GitHub actions.

**Comment by jeesang7** [comment]
Did you solve this issue?
If it's an NDK version problem, you can refer to the following.

https://github.com/subosito/flutter-action/issues/116

Also, I wonder if the execution results of other OS environments (Windows, macOS) are the same.
